src/GeoInterfaceRFC.jl

```diff
@@ -5,4 +5,26 @@ include("interface.jl")
 include("defaults.jl")
 include("primitives.jl")
 
+export
+Point
+LineString
+Line
+LinearRing
+CircularString
+CompoundCurve
+CurvePolygon
+Polygon
+Triangle
+Rectangle
+Quad
+Pentagon
+Hexagon
+PolyHedralSurface
+TIN
+MultiPoint
+MultiCurve
+MultiLineString
+MultiPolygon
+GeometryCollection
```

Comment:
I'm thinking, since these types are only used as traits and are not otherwise usable, maybe we should not export them? This will also clash with for instance several of the exports from GeometryBasics.

Reply:
That's a good point. I do assume we export all other functions? Like ngeom, ncoord, etc.

Reply:
Not sure about 'all', but perhaps those that are expected to be used by end users rather than interface implementers?